src/exercise5.js

```diff
@@ -0,0 +1,17 @@
+function dec2bin(num) {
+  return (num >>> 0).toString(2);
+}
+console.log(dec2bin(3));
+
+const fibonnaciSequence = (num)=>{
```

Comment:
Also as we discussed if possible a function should be named from an action (a verb) it is doing